ui/frontend/editor/MonacoEditorCore.tsx

```diff
@@ -1,17 +1,16 @@
 import React from 'react';
 import { CommonEditorProps } from '../types';
-import MonacoEditor, { EditorWillMount } from 'react-monaco-editor';
+import MonacoEditor, { EditorDidMount, EditorWillMount } from 'react-monaco-editor';
 import { useSelector } from 'react-redux';
 import State from '../state';
 import { config, grammar, themeVsDarkPlus } from './rust_monaco_def';
 
 import styles from './Editor.module.css';
 
-const MODE_ID = 'my-rust';
+const MODE_ID = 'rust';
```

Comment:
Why this change?

Reply:
Somethings (like hover snippets) can not be set to use `my-rust`. Not related to the issues, but still something that isn't correct.